starter-core/src/main/java/io/micronaut/starter/feature/test/MicronautRestAssured.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.test;
+
+import io.micronaut.starter.application.ApplicationType;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.build.dependencies.MicronautDependencyUtils;
+import io.micronaut.starter.feature.Category;
+import io.micronaut.starter.feature.Feature;
+import jakarta.inject.Singleton;
+
+@Singleton
+public class MicronautRestAssured implements Feature {
+    private static final String ARTIFACT_ID_MICRONAUT_TEST_REST_ASSURED = "micronaut-test-rest-assured";
+
+    @Override
+    public String getName() {
+        return "micronaut-test-rest-assured";
+    }
+
+    @Override
+    public String getTitle() {
+        return "Micronaut-Test Rest-Assured";
```

Comment:
Replace `Rest-Assured` with `REST-assured`. That it is how they write it. see: https://rest-assured.io